corehq/apps/translations/app_translations/upload_form.py

```diff
@@ -85,6 +86,9 @@ def update(self, rows):
         # Skip labels that have no translation provided
         label_ids_to_skip = self._get_label_ids_to_skip(rows)
 
+        if self.submit_label:
+            self._update_translation(self.submit_label, self.form.submit_label)
```

Comment:
Could you tell me a little about how this works. I'm looking at `self.markdowns` and `self.markdown_vetoes` and they seem to be updated with `_update_translation_node`.

Is this referencing the module method?
https://github.com/dimagi/commcare-hq/blob/df41a25481b6a2ab350d3af6ca1eff82d22d6ecf/corehq/apps/translations/app_translations/upload_module.py#L307

Reply:
@Robert-Costello ah yes this is updated now-- see if the new logic makes more sense